dev/io.openliberty.concurrent.internal/resources/io/openliberty/concurrent/internal/resources/CWWKCMessages.nlsprops

```diff
@@ -0,0 +1,45 @@
+#ISMESSAGEFILE TRUE
+#NLS_ENCODING=UNICODE
+#
+#COMPONENTPREFIX CWWKC
+#COMPONENTNAMEFOR CWWKC Jakarta EE Concurrency
+#
+# NLS_MESSAGEFORMAT_VAR
+#
+#   Strings in this file which contain replacement variables are processed by the MessageFormat 
+#   class (single quote must be coded as 2 consecutive single quotes ''). Strings in this file 
+#   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
+#   (single quote must be coded as one single quote '). 
+
+# All messages must use the range CWWCK1200 to CWWCK1399
+
+CWWKC1200.resource.unavailable=CWWKC1200E: {0}, which is defined by application {1}, did not become available in a timely manner.
+CWWKC1200.resource.unavailable.explanation=There might be too much load on the system to process resource definition annotations and deployment descriptor entries before the application attempts to use them.
+CWWKC1200.resource.unavailable.useraction=View the logs to identify if an error occurred. Otherwise, reduce the load on the system.
+
+CWWKC1201.contextsvc.inaccessible=CWWKC1201E: {0}, which is defined by application {1}, requires a ContextService {2} but the ContextService is inaccessible or did not become available in a timely manner.
+CWWKC1201.contextsvc.inaccessible.explanation=A resource definition annotation or a resource definition in a deployment descriptor of the application specifies an incorrect JNDI name at which to find a ContextService, or the ContextService is defined in a scope which is not accessible to the resource, or the system is under too much load to register the ContextService in a timely manner.
+CWWKC1201.contextsvc.inaccessible.useraction=Correct the application to refer to the JNDI name of ContextService that is accessible. Reduce the load on the system if the JNDI name is already correct.
+
+CWWKC1202.context.lists.overlap=CWWKC1202E: Conflicting configuration of thread context {0} is found on ContextServiceDefinition {1}. Cleared context is {2}. Propagated context is {3}. Unchanged context is {4}.
+CWWKC1202.context.lists.overlap.explanation=A thread context type can be configured to any one of: cleared, propagated, or left unchanged. The context type cannot be configured to be in multiple context type categories.
+CWWKC1202.context.lists.overlap.useraction=Correct the configuration to specify the thread context type in at most one category.
+
+CWWKC1203.duplicate.context=CWWKC1203E: The same thread context type, {0}, is provided by multiple thread context providers which are available to the application. Thread context providers are: {1}, {2}.
+CWWKC1203.duplicate.context.explanation=There can only be one ThreadContextProvider for each context type.
+CWWKC1203.duplicate.context.useraction=Update the application, libraries, or both so that no more than one ThreadContextProvider of the specified context type is accessible to the application.
+
```

Comment:
Reviewed Messages:
CWWKC1203.duplicate.context=CWWKC1203E: The same thread context type, {0}, is provided by multiple thread context providers that are available to the application. The thread context providers are: {1}, {2}.

CWWKC1203.duplicate.context.explanation=There can only be one ThreadContextProvider for each context type.
[Previously you have used thread context providers, here it is **ThreadContextProvider** should it be mentioned as one word?]

CWWKC1203.duplicate.context.useraction=Update the application, libraries, or both so that no more than one ThreadContextProvider of the specified context type is accessible to the application.

Reply:
ok